l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
@renatonlima because the field date_hour_invoice is declared in l10n_br_account_product. But if using date_invoice to order, which now I think should be the best, the l10n_br_account_product  _order can be removed.